### PR TITLE
Cleaning up the README

### LINKED ISSUE #640: Should these 2 dependencies be mentioned somewhere?
Hi,

I found 2 issues while compiling this this project. The compilation process aborted twice due to missing `cmake` and `libxml2-dev`. I am using Ubuntu 18.04.

### PR DESCRIPTION
**Pull request by jwoertink**
Fixes #640

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,18 @@
 # Lucky's Website
 
-### Future guide goals
-
-- Add screencasts
-- _Versioned_. Look at old ones, or even look at what's on master
-- Link to relevant screencasts
-- Helpful or not button
-  - If not, direct to Discord to get help
-- Check code snippets in real projects. More confidence code samples work.
```
Comment (jwoertink) [inline README.md:10]: we have screencasts linked through LuckyCasts, and some of these are already created as issues.

**Review by stephendolan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,24 +1,18 @@
...
+This is the repository for Lucky's official website https://luckyframework.org/
```
Comment (hovsater) [inline README.md:3]: ending `.` perhaps?

File: README.md [hunk 3]
```diff
@@ -1,24 +1,18 @@
...
 1. [Install required dependencies](https://luckyframework.org/guides/getting-started/installing#install-required-dependencies)
+1. Ensure you have `cmake` and `libxml2` installed for your system. These are required for some shard dependencies of this project
 1. Run `script/setup`
 1. Run `lucky dev` to start the app
 1. Visit `http://localhost:5000`
 
 ## Editing guides
 
-Guides are located in the `src/actions/guides`. You can edit the markdown in each action file and view changes by running the Lucky app with `lucky dev`
+Guides are located in the `src/actions/guides` directory. You can edit the markdown in each action file and view changes by running the Lucky app with `lucky dev`.
```
Comment (hovsater) [inline README.md:15]:
```suggestion
Guides are located in the `src/actions/guides` directory. You can edit the Markdown in each action file and view changes by running the Lucky app with `lucky dev`.
```

**Review by hovsater — APPROVED** [review]
LGTM. 🚢

**Review by matthewmcgarvey — APPROVED** [review]

**Pull request merged**